build.env

```diff
@@ -16,12 +16,12 @@ BASE_IMAGE=docker.io/ceph/ceph:v15
 CEPH_VERSION=octopus
 
 # standard Golang options
-GOLANG_VERSION=1.15.5
+GOLANG_VERSION=1.16.2
```

Comment:
this is not a static check tool. If an update of Go is required, explain the reason in the commit message.

Reply:
okay.